test/transfer.test.ts

```diff
@@ -0,0 +1,116 @@
+import { describe, test, afterEach, before } from "mocha";
+import { spy, restore, assert, match } from "sinon";
+import yargs from "yargs/yargs";
+import { getAccounts } from "@tableland/local";
+import * as mod from "../src/commands/transfer.js";
+import { wait } from "../src/utils.js";
+import { helpers } from "@tableland/sdk";
+
+describe("commands/transfer", function () {
+  this.timeout("30s");
+
+  before(async function () {
+    await wait(10000);
+  });
+
+  afterEach(function () {
+    restore();
+  });
+
+  test("throws without privateKey", async function () {
+    const consoleError = spy(console, "error");
+    await yargs(["transfer", "healthbot_31337_1", "0x0000000000000000000000"])
+      .command(mod)
+      .parse();
+    assert.calledWith(
+      consoleError,
+      "No registry. This may be because you did not specify a private key with which to interact with the registry."
+    );
+  });
+
+  test("throws with invalid chain", async function () {
+    const [account] = getAccounts();
+    const privateKey = account.privateKey.slice(2);
+    const consoleError = spy(console, "error");
+    await yargs([
+      "transfer",
+      "healthbot_337_1",
+      "0x0000000000000000000000000000000000000000",
+      "--privateKey",
+      privateKey,
+    ])
+      .command(mod)
+      .parse();
+    assert.calledWith(
+      consoleError,
+      "unsupported chain (see `chains` command for details)"
+    );
+  });
+
+  test("throws with invalid table name", async function () {
+    const [account] = getAccounts();
+    const privateKey = account.privateKey.slice(2);
+    const consoleError = spy(console, "error");
+    await yargs(["transfer", "healthbot", "blah", "-k", privateKey])
+      .command(mod)
+      .parse();
+    assert.calledWith(
+      consoleError,
+      "invalid table name (name format is `{prefix}_{chainId}_{tableId}`)"
+    );
+  });
+
+  test("throws with invalid receiver address", async function () {
+    const [account] = getAccounts();
+    const privateKey = account.privateKey.slice(2);
+    const consoleError = spy(console, "error");
+    await yargs([
+      "transfer",
+      "healthbot_31337_1",
+      "0x00",
+      "--privateKey",
+      privateKey,
+      "--chain",
+      "local-tableland",
+    ])
+      .command(mod)
+      .parse();
+    assert.calledWith(
+      consoleError,
+      'invalid address (argument="address", value="0x00", code=INVALID_ARGUMENT, version=address/5.7.0)'
+    );
+  });
+
+  // Does transfering table have knock-on effects on other tables?
```

Comment:
This test actually transfers a table on local-tableland. @joewagner Should I transfer the table back at the end of the test? I don't think any tests depend on the owner of the table being account1, but I'm not sure.

Reply:
I'll take a closer look, but in general, I've been trying to creating a table for the test instead of using healthbot's table when possible.

Reply:
@awmuncy something like this is what I was thinking https://github.com/tablelandnetwork/js-tableland-cli/commit/1aae786888c277dcb2521714eafc6e5041725beb

Reply:
@joewagner Beautiful, pulled that in.